fastly/resource_fastly_service_v1.go

```diff
@@ -1385,6 +1385,26 @@ func resourceServiceV1() *schema.Resource {
 					},
 				},
 			},
+			"acl": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// Required fields
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Unique name to refer to this ACL",
+						},
+						// Optional fields
+						"acl_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Generated acl id",
+						},
+					},
+				},
+			},
```

Comment:
The `acl` nested resource needs to be documented in the `fastly_service_v1` documentation. Please also think about if the `acl_id` is an exported attribute and should be listed under https://www.terraform.io/docs/providers/fastly/r/service_v1.html#attributes-reference.

Reply:
The ACL nest block was documented and the acl_id was listed under the exported attributes.  This will help to understand the use in the import to the entries resource.